### ISSUE TITLE
Solving environment: failed with initial frozen solve. Retrying with flexible solve. Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.

### ISSUE THREAD

**Issue by sanweishuwu100** [AutodeskAILab/occwl#26]
When I use the method on the right（conda install -c lambouj -c conda-forge occwl）, the computer will show that the error is     
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source. 
Can someone help me? Thank you.

**Comment by JoeLambourne** [comment]
Hi @sanweishuwu100,

We are now getting the same issue here.   I thought this was related to #21 and I released a new package with the specific version of `ipykernel` removed.   I tried another test install but it's still hanging like this

```
~/projects$ conda activate joe_testing_occwl
(joe_testing_occwl) ~/projects$ conda install -c lambouj -c conda-forge occwl
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: -
```
I guess this is what you are seeing as well.   @pradeep-pyro had a similar issue and had to work around using mamba 

https://mamba.readthedocs.io/en/latest/

I'm not sure what could have changed to cause the issue.   I can see install the underlying pythonocc without an issue

**Comment by pradeep-pyro** [comment]
Yes, I was running into similar issues in other environments that had `occwl`, and tried using mamba.
What I did is:
```
conda install -n base mamba -c conda-forge
...
mamba install occwl -c lambouj -c conda-forge
```
and the installation went through instantly.

I am not sure what the root cause of this issue is yet, but I suspect it might be on the `conda` side and related to dependency resolution.

**Comment by JoeLambourne** [comment]
@sanweishuwu100,

Please do let us know if this fixed your issue.   If so then I would like to update the documentation so others don't hit the same problems...
